### PR TITLE
product-page

### PR DESCRIPTION
**Pull request by talkalover**

### CODE REVIEW

File: src/components/product.vue [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      <img :src="product.img" imgfield="img" style="max-width: 150px; width: 150px;" alt=""> 
+      <p>{{ product.name }}</p> 
+    </div>
+
+  
+
+    <div class="logo">
+   <img src="https://static.tildacdn.info/tild6431-3661-4430-b735-353863373435/11.svg" imgfield="img" style="max-width: 150px; width: 150px;" alt=""> 
+  </div>
+<div class="headline">Мы любим создавать особенные и уникальные вещи  <br/>для украшения вашей жизни и дома </div>
```
Comment (riffca) [inline src/components/product.vue:15]: rewrite list iof products using v-for and objects

File: src/components/product.vue [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  {
+    img: 'https://static.tildacdn.info/tild6431-3661-4430-b735-353863373435/11.svg',
+    price: 30,
+    name: 'Red nose'
+  }
+]
+
+</script>
+
+<style lnag="scss">
```
Comment (riffca) [inline src/components/product.vue:104]: use scoped attribute. Means that in html we will have unique classnames for example `logo-3fd5as`

File: src/components/product.vue [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+.logo{
```
Comment (riffca) [inline src/components/product.vue:105]: use more complicated names. This is `products_page` so it should be `products_page--logo`

File: src/components/product.vue [hunk 4]
```diff
@@ -0,0 +1,163 @@
+<template>
```
Comment (riffca) [inline src/components/product.vue:1]: move products list to `products` component
move remaining html and products component to products page

Comment (riffca) [reply src/components/product.vue:1]: use props check issue #4

**Comment by talkalover** [comment]
rename branch to products-page

File: src/components/products-list.vue [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<template> 
+<div :class="'product'+(index+1)" v-for="product, index in products"> 
+      <img :src="product.img" imgfield="img" style="max-width: 150px; width: 150px;" alt=""> 
+      <p>{{ product.name }}</p> 
+    </div>
+</template>
+<script setup lang="ts">
+
+type Product =  {
```
Comment (riffca) [inline src/components/products-list.vue:9]: Move type Product to stores/product.ts

import it to product page and product-list 
`import type { Product } from '@/stores/product`

**Pull request closed without merging**